Lib/test/test_asyncio/test_events.py

```diff
@@ -2156,6 +2156,8 @@ def tearDown(self):
             super().tearDown()
 
         def test_get_event_loop_new_process(self):
+            # Skip the test if the sem_open() implementation is broken.
+            support.import_module('multiprocessing.synchronize')
```

Comment:
The check is very non-obvious.
Don't how to make it more verbose and explicit.
If it's impossible -- I can live with it.

Reply:
The ``ProcessPoolExecutor`` uses the ``multiprocessing`` (``mp``) module. The ``mp`` module is not functional when the ``mp.synchronize`` module cannot be imported and all the ``mp`` tests are skipped in that case. So we use here the same idiom used to skip the ``mp`` tests (i.e. ``test.support.import_module('multiprocessing.synchronize')``) to skip this test when the platform ``sem_open`` implementation is broken or missing.

What about replacing the comment with:
```
Issue bpo-32126: The multiprocessing module used by ProcessPoolExecutor
is not functional when the multiprocessing.synchronize module cannot be imported.
```

Reply:
New comment text is perfect!